addons/mail/models/mail_link_preview.py

```diff
@@ -87,39 +87,50 @@ def _is_domain_throttled(self, url):
         return call_counter > link_preview_throttle
 
     @api.model
-    def _get_link_preview_from_url(self, url, request_session):
+    def _get_link_preview_from_url(self, url, request_session=None):
         try:
-            response = request_session.head(url, timeout=3, allow_redirects=True)
+            # Some websites are blocking non browser user agent.
+            user_agent = {'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; rv:91.0) Gecko/20100101 Firefox/91.0'}
+            if request_session:
+                request_session.headers.update(user_agent)
+                response = request_session.get(url, timeout=3, allow_redirects=True, stream=True)
+            else:
+                response = requests.get(url, timeout=3, headers=user_agent, allow_redirects=True, stream=True)
         except requests.exceptions.RequestException:
             return False
-        if response.status_code != requests.codes.ok:
-            return False
-        image_mimetype = (
-            'image/bmp',
-            'image/gif',
-            'image/jpeg',
-            'image/png',
-            'image/tiff',
-            'image/x-icon',
-        )
```

Comment:
Are you sure this is acceptable?
The goal was to avoid svg from being considered for security reasons.

Reply:
Also our image lib does not work with other formats anyway.

Reply:
Mmmm now that I think about it, in the final version of link preview it looks like we only store the url and not the image, so it's probably acceptable to make this diff.